### ISSUE TITLE
package updates break config

### ISSUE THREAD

**Issue by matt-long** [NCAR/intake-esm#47]
Perhaps this won't be an issue once things settle down...but issues arise when a user updates `intake-esm` to a version that changed the `config.yaml`. I don't see an easy way to deal with this.

**Comment by matt-long** [comment]
One thing that might help would be to separate `config.yaml` into two files: `config-collections.yaml` and `config-user.yaml`.  At this point, the latter would contain 
```yaml
default_chunk_size: 128MiB
data_cache_directory: ~/.intake_esm/data_cache
database_directory: ~/.intake_esm/collections
```
Having the default cache directory in $HOME is dangerous if we ever get to the point of using that space to cache files. Quotas on $HOME would be rapidly exceeded on systems like Cheyenne.

**Comment by andersy005** [comment]
Addressed in #83
